lib/editor/util/validation.js

```diff
@@ -31,7 +31,7 @@ export function validate (
   entities: ?Array<Entity>,
   entity: ?Entity,
   tableData: EditorTables
-): EditorValidationIssue | false {
+): EditorValidationIssue | false | Array<EditorValidationIssue> {
```

Comment:
Is there any way to make this always return an array? I think the single value/array union type will cause more problems than it solves.

Reply:
We could, but that would mean modifying the `validate` method to always return an array. Because the method handles a bunch of different validation cases it has a lot of returns and this would be a change in each one of those places. Happy to make that change, just lmk what you think

Reply:
Ah makes sense. Let's leave it as is then!